I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/custom/login.rpy", line 38, in script
    $input_error()
  File "game/custom/login.rpy", line 38, in <module>
    $input_error()
  File "game/globalCommands.rpy", line 73, in input_error
    term("{cps=125}'[" + s + "]': {color=#f00}command not found{/color}.  Type 'help' or '?' for available commands.")
KeyError: u'asdf'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/custom/login.rpy", line 38, in script
    $input_error()
  File "/Applications/renpy-6.99.12.4-sdk/renpy/ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-6.99.12.4-sdk/renpy/python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/custom/login.rpy", line 38, in <module>
    $input_error()
  File "game/globalCommands.rpy", line 73, in input_error
    term("{cps=125}'[" + s + "]': {color=#f00}command not found{/color}.  Type 'help' or '?' for available commands.")
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 868, in __call__
    what = what_pattern.replace("[what]", sub(what, translate=translate))
  File "/Applications/renpy-6.99.12.4-sdk/renpy/character.py", line 850, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "/Applications/renpy-6.99.12.4-sdk/renpy/substitutions.py", line 232, in substitute
    s = formatter.vformat(s, (), kwargs)
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 563, in vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 585, in _vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 646, in get_field
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 605, in get_value
KeyError: u'asdf'

Darwin-16.7.0-x86_64-i386-64bit
Ren'Py 6.99.12.4.2187
> turing 0.1
